packages/e2e-test-utils/src/inserter.js

```diff
@@ -41,7 +41,7 @@ export async function closeGlobalBlockInserter() {
 async function isGlobalInserterOpen() {
 	return await page.evaluate( () => {
 		return !! document.querySelector(
-			'.edit-post-header [aria-label="Toggle block inserter"].is-pressed, .edit-site-header [aria-label="Toggle block inserter"].is-pressed'
+			'.edit-post-header [aria-label="Add block"].is-pressed, .edit-site-header [aria-label="Add block"].is-pressed, .edit-post-header [aria-label="Toggle block inserter"].is-pressed, .edit-site-header [aria-label="Toggle block inserter"].is-pressed'
```

Comment:
Can we add a comment explaining why we need both, so future contributors can make good decisions about when to remove.